files/bash-completion

```diff
@@ -31,7 +31,12 @@ __fail2ban_jail_action_methods () {
 
 _fail2ban () {
     local cur prev words cword
-    _init_completion || return 
+    _init_completion || return
+
+    if [[ "$1" == *"fail2ban-regex" ]] && [[ "$cur" == "-"* ]];then
```

Comment:
with this change it doesn't suggest any file for the first argument `<LOG>` and would only complete options (but that also did it before).  So what gets fixed really? ;)

PS sorry for the delay